### DISCUSSION TITLE
Some resources do not require tenant scope queries

### DISCUSSION THREAD

**Discussion by pslxx** [filamentphp/filament#8336] [Help]
Some resources do not require tenant scope queries

Because it is a public resource and is used by the super administrator

What should I do```` need help

**Comment by pslxx** [comment]
ScopeEloquentQueryToTenant  is only useful in lists, and there may still be issues when creating and editing them

**Reply by decopiller** [reply]
You can use trait on boot when creating and updating

**Comment by pslxx** [comment]
i wish can add a property like as 

$skipTenant = true or $ignoreTenant = true    thats perfect

**Comment by pslxx** [comment]
need help !
